apps/console/src/public/deployment.config.json

```diff
@@ -315,7 +315,7 @@
         "isGroupAndRoleSeparationEnabled": true,
         "isSignatureValidationCertificateAliasEnabled" : false,
         "isClientSecretHashEnabled": false,
-        "isDefaultDialectEditingEnabled": true,
+        "isDefaultDialectEditingEnabled": false,
```

Comment:
If you need to make the dialect URI readonly by default, you'll have to an entry in the [default.json](https://github.com/brionmario/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json). Since this entry is set to defined only if it is overrided by toml. (https://github.com/wso2/identity-apps/blob/master/features/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2#L502)

Reply:
done https://github.com/wso2/carbon-identity-framework/pull/3358

Reply:
👍